tests/lowest_rate_test.go

```diff
@@ -0,0 +1,75 @@
+package easypost_test
+
+import (
+	"github.com/EasyPost/easypost-go"
+)
+
+func GenerateTestShipment() *easypost.Shipment {
+
```

Comment:
Don't open a function with a line-break, it should immediately jump into the code.